ckan/logic/action/get.py

```diff
@@ -323,7 +323,12 @@ def _group_or_org_list(context, data_dict, is_org=False):
         if errors:
             raise ValidationError(errors)
     sort = data_dict.get('sort') or config.get_value('ckan.default_group_sort')
-    q = data_dict.get('q')
+    k = data_dict.get('q')
+    if k == None:
+        q = k
+    else:
+        q = k.strip()
```

Comment:
@Sunny-NEC this is a bit more readable:

```suggestion
    q = data_dict.get('q')
    if q:
        q = q.strip()
```

Or even
```suggestion
    q = data_dict.get('q', '').strip()
```

Reply:
@amercader update the file  according to your suggestion. if it looks good . merge this PR.